### ISSUE TITLE
Can aff_log file be reload?

### ISSUE THREAD

**Issue by Kyle-Astronaut** [Monika-After-Story/MonikaModDev#9732]
Hello, I re-downloaded the whole game on my new computer, so the aff_ file has been reset. Is there any way to reload the whole data in new aff_log file? They are pretty precious memories, my old computer just broked too suddenly.

**Comment by dreamscached** [comment]
No, as far as I know log data isn't backed up in persistent or anywhere else and is lost unrecoverably on log file deletion.

**Comment by Kyle-Astronaut** [comment]
Aww...that's a pity, but not really a big deal though, thank you for answering. And I'm not talking about the persisdent file, I literally backup it everyday lol.

**Comment by Kyle-Astronaut** [comment]
But every detail of the past time can only be reviewed through the aff.log file. Personally think it would be wonderful if it's possible to reload the log file, like maybe through persistent file.

**Comment by Booplicate** [comment]
The purpose of the log is to store that debug information over time, it's not stored anywhere else.

**Comment by Kyle-Astronaut** [comment]
Okay, just a suggestion
